From d6ede3a756010bf332c74526bf6bf670d6ef3646 Mon Sep 17 00:00:00 2001
From: Emily Deng <Emily.Deng@amd.com>
Date: Thu, 9 Aug 2018 15:05:31 +0800
Subject: [PATCH 5447/5725] drm/amdgpu/sriov: Correct the setting about sdma
 doorbell offset of Vega10

Correct the format

For vega10 sriov, the sdma doorbell must be fixed as follow to keep the
same setting with host driver, or it will happen conflicts.

Change-Id: I0e7480c6516a4fe4c2646f1644f897e21e0f7f7b
Signed-off-by: Emily Deng <Emily.Deng@amd.com>
Acked-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Felix Kuehling <Felix.Kuehling@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu.h b/drivers/gpu/drm/amd/amdgpu/amdgpu.h
index 80fc9b3..7d10f6e 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu.h
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu.h
@@ -426,8 +426,9 @@ typedef enum _AMDGPU_DOORBELL64_ASSIGNMENT
 	AMDGPU_DOORBELL64_sDMA_HI_PRI_ENGINE1     = 0xE9,
 
 	/* For vega10 sriov, the sdma doorbell must be fixed as follow
-	* to keep the same setting with host driver, or it will
-	* happen conflicts */
+	 * to keep the same setting with host driver, or it will
+	 * happen conflicts
+	 */
 	AMDGPU_VEGA10_DOORBELL64_sDMA_ENGINE0            = 0xF0,
 	AMDGPU_VEGA10_DOORBELL64_sDMA_HI_PRI_ENGINE0     = 0xF1,
 	AMDGPU_VEGA10_DOORBELL64_sDMA_ENGINE1            = 0xF2,
-- 
2.7.4

